Makefile

```diff
@@ -134,33 +134,34 @@ JS_DEPS=$(JS)/vendor/jquery-latest.min.js \
 
 
 JS_SOURCE=$(JS)/respond.js \
-	$(JS)/variables.js \
-	$(JS)/utils.js \
-	$(JS)/ermrest.js \
 	$(JS)/app.js \
-	$(JS)/facetsModel.js \
-	$(JS)/facetsService.js \
 	$(JS)/controller/ermrestDetailController.js \
 	$(JS)/controller/ermrestFilterController.js \
 	$(JS)/controller/ermrestInitController.js \
 	$(JS)/controller/ermrestLoginController.js \
 	$(JS)/controller/ermrestResultsController.js \
 	$(JS)/controller/ermrestSideBarController.js \
 	$(JS)/controller/ermrestTourController.js \
+	$(JS)/ermrest.js \
```

Comment:
I was organizing them better and didn't realize I changed the `JS_RESOURCE` at first. The order does matter for some of the resources though (jquery needs to be defined first before angular for instance).